### ISSUE TITLE
contact us form

### ISSUE THREAD

**Issue by mkaf89** [callummac88/bbf#4]
"

Someone has sent you a message using your contact form:

Name: Charles Brusatori
Email: Guitarchaz67@hotmail.com
Subject: Someone has sent you a message for banchory beer festival
"

When we get emails from the website it comes from the following email address "callumma@serv01.ams12.siteground.eu" would it be possible to set it up to come from the person who's contact us email address. It's not to much of an issue now but when the mass of morons start emailing its a lot quicker just to hit reply then having to also change the reply to address.?

**Comment by mkaf89** [comment]
Same issues on IBF- also would it be possible to go to a page saying email sent rather than directing to the home page?

**Comment by mkaf89** [comment]
any updates on this?

**Comment by callummac88** [comment]
I dont think we can change this (not easily anyways) as its actually the email functionalility of the server that sends the mail, not the users own client. It means anyone connected to the internet can use the form, they dont need to be set up with a local client

**Comment by mkaf89** [comment]
What about a different plugin or something like- 

http://www.smartaddon.com/contact_form.html

http://www.123contactform.com/html-contact-form/

or does that still use the servers email address?

**Comment by mkaf89** [comment]
There must be some work around on this as all the other website ive got inquiry forms on come from a dedicated email address.

**Comment by mkaf89** [comment]
any update on this? needing all this issues closed asap.

**Comment by callummac88** [comment]
when you say 'come from a dedicated email address' what do you mean?

**Comment by mkaf89** [comment]
So it would come from say info@banchorybeerfestival.com email address and it would have a reply-to email address which would be the persons email address who has enquired rather than the callum@server email address and i then have to replace that email address with the email address of the person whos enquired. Get my drift?
